client/blocks/jitm/index.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSite } from 'state/ui/selectors';
+import Banner from 'components/banner';
+
+const unescape = ( str ) => {
```

Comment:
Is this a generic text escape? If so, is it available in a more generic library? If not, should it be?

Reply:
There's something similar in `lodash` but it won't get all strings. Anything more than this one liner is akin to requiring `left-pad` ... my 2¢

Reply:
what strings "won't it get"?

also, one good reason for these kinds of libraries is that they can handle a wide range of specs. for example, it appears like this function is supposed to replace escaped HTML entities with their Unicode equivalents, but what about `&gt;` and friends? or `&#x5d0`?

actually it's great to be pragmatic and not bring in a bunch of code, but we should at least indicate what exactly are goals are in places like this, maybe with a name like `replaceIntegerEntities()`.

feel free to use `lodash` - it's already standard in the codebase and already loaded so using it won't bring in any more code

Reply:
@withinboredom concerning the utils I wouldn't worry about a function this small with just two copies. it will honestly likely be _more_ code and _mode_ duplication to extract it into a utils library. will that matter? not a whole lot.

I'm more concerned about knowing what the purpose of this function is, and even more so (looking at my comments below on the API interface) I'm concerned about its placement here in the component itself, which I would expect to just take a string and not have to think about escaping